docs/grammar/SolidityLexer.g4

```diff
@@ -111,7 +111,8 @@ Colon: ':';
 Semicolon: ';';
 Period: '.';
 Conditional: '?';
-Arrow: '=>';
+DoubleArrow: '=>';
+RightArrow: '->';
```

Comment:
This is here for the lexer to pick it up.

Reply:
Does this have implications for the AST output? If Yes, I would prefer not changing `Arrow`, but it certainly has to be mentioned in its own changelog entry.

Reply:
It is an internal detail, isn't it? How would the AST be affected? None of the AST test emit a difference, but that could just mean tests are bad.